src/zenml/integrations/aws/flavors/sagemaker_orchestrator_flavor.py

```diff
@@ -30,6 +30,9 @@
 if TYPE_CHECKING:
     from zenml.integrations.aws.orchestrators import SagemakerOrchestrator
 
+DEFAULT_TRAINING_INSTANCE_TYPE = "ml.m5.xlarge"
```

Comment:
Might be nice to mention these default types in the docs as well.

Reply:
done